hack/lib/init.sh

```diff
@@ -115,6 +115,7 @@ scheduling.k8s.io/v1 \
 storage.k8s.io/v1beta1 \
 storage.k8s.io/v1 \
 storage.k8s.io/v1alpha1 \
+storagemigration.k8s.io/v1alpha1 \
 flowcontrol.apiserver.k8s.io/v1beta1 \
```

Comment:
Very obvious in this context that the group naming is unfortunate. Had a similar comment on the KEP I believe, but got ignored.

Reply:
I'm open to suggestions.

Reply:
with some kind like `ResourceMigration` that doesn't stutter.